pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -94,6 +94,9 @@
   hoc_overview_csc_header: 'Introducing CS Connections'
   hoc_overview_csc_desc: 'Our newest curriculum highlights the connections between computer science and other subjects like math, language arts, science, and social studies. Through these modules, projects, and lessons, classrooms can explore their usual subjects in exciting new ways!'
   hoc_overview_csc_learn_more: 'Learn more about CS Connections'
+  hoc_danceparty_alt_text: 'gif of characters dancing underneath the text, Code your own dance party'
+  hoc_ai_alt_text: 'Illustration of AI bot'
```

Comment:
How about "A floating, white, artificial intelligence character with an antennae, a scanner on the bottom, and an LED face"